packages/client/src/actions/vaults.test.ts

```diff
@@ -448,6 +450,21 @@ describe('Given the Aave Vaults', () => {
         assertOk(redeemResult);
       });
 
+      const timeWindows = Object.values(VaultUserActivityTimeWindow);
+      it.each(timeWindows)(
```

Comment:
if you are creating new vault and deposit a known value than we could also check math logic as it won't really change between reruns